filebeat/README.md

```diff
@@ -0,0 +1,20 @@
+# Elastic Enterprise Search Filebeat configuration
+
+This directory contains Filebeat configuration to be used with [Filebeat](https://www.elastic.co/guide/en/beats/filebeat) in order to ingest Enterprise Search log files into Elasticsearch.
+
+This example configuration assumes Filebeat and Elasticsearch versions 8.x  
+
+## Installation
+
+1. [Download Filebeat](https://www.elastic.co/downloads/beats/filebeat)
+2. Replace `filebeat.yml` file in the Filebeat install directory with the one in this directory
```

Comment:
3. Copy `filebeat-template-8.json` from this directory into your Filebeat install directory.